platform/src/test/java/org/hillview/test/storage/SSTableTest.java

```diff
@@ -53,11 +53,21 @@ public void testSSTableComplimentaryFiles() {
     @Test
 
     public void testReadingSSTable() {
-        CassandraSSTableLoader ssTableLoader = new CassandraSSTableLoader(this.ssTablePath, false);
+        boolean lazyLoading = false;
+        CassandraSSTableLoader ssTableLoader = new CassandraSSTableLoader(this.ssTablePath, lazyLoading);
         try {
             ITable table = ssTableLoader.load();
+            String name = table.getLoadedColumn("name").getString(6);
+            String address = table.getLoadedColumn("address").getString(6);
+            int salary = table.getLoadedColumn("salary").getInt(6);
+            String phone = table.getLoadedColumn("phone").getString(6);
+
             Assert.assertNotNull(table);
             Assert.assertEquals("Table[4x15]", table.toString());
+            Assert.assertEquals("azam", name);
+            Assert.assertEquals("Chennai", address);
+            Assert.assertEquals(45000, salary);
+            Assert.assertEquals("9848022330", phone);
```

Comment:
If you checked-in this file there is no reason this test should fail, so perhaps you can remove all these catch

Reply:
Right, I will remove them